src/components/SearchPerson/SearchPerson.tsx

```diff
@@ -1,16 +1,184 @@
 import * as React from 'react'
-//import { gql } from '@apollo/client'
-import { Row, Col } from 'react-bootstrap'
+import { Row, Alert } from 'react-bootstrap'
+import { gql, useQuery } from '@apollo/client'
+import { useQueryState } from 'next-usequerystate'
+// eslint-disable-next-line no-unused-vars
+import {  PersonType } from "../PersonContainer/PersonContainer"
+import Pager from "../Pager/Pager"
+import ContentLoader from "react-content-loader"
+import Error from "../Error/Error"
+import PersonMetadata from "../PersonMetadata/PersonMetadata"
+
 
 type Props = {
     searchQuery: string;
 };
 
+
+interface ContentQueryData {
+    people: {
+      __typename: String
+      nodes: PersonType[]
+      pageInfo: PageInfo
+    },
+  }
+
+interface PageInfo {
+    endCursor: string
+    hasNextPage: boolean
+}
+  
+interface ContentQueryVar {
+    query: string
+    cursor: string
+}
+
+export const CONTENT_GQL = gql`
+  query getContentQuery($query: String, $cursor: String) {
+    people(first: 25, query: $query, after: $cursor) {
+    nodes {
+      id
+      name
+      givenName
+      familyName
+      # citationCount
+      # viewCount
+      # downloadCount
+      affiliation {
+        name
+        id
+      }
+      works{
+        totalCount
+      }
+    }
+  }
+}
+`
+
 const SearchPerson: React.FunctionComponent<Props> = ({ searchQuery }) => {
+    // const router = useRouter()
+
+    /* eslint-disable no-unused-vars */
```

Comment:
Are these really unused? We have to be very careful about ignoring es-lint errors.